tests/test_glue/test_parser.xml

```diff
@@ -74,6 +78,7 @@
                     glue("key_plt", plt.gcf(), display=False)
             <CellOutputNode classes="cell_output">
                 <image candidates="{'*': '_build/jupyter_execute/with_glue_5_1.png'}" uri="_build/jupyter_execute/with_glue_5_1.png">
+                <image candidates="{'*': '_build/jupyter_execute/with_glue_5_1.png'}" uri="_build/jupyter_execute/with_glue_5_1.png">
```

Comment:
same as above, why are outputs being duplicated?

Reply:
it might have something to do with jupyter cache 🤷🏻

Reply:
there was duplicate scrapbook metadata causing the output to show twice. this is resolved by popping scrapbook from the new metadata.